rss-client/src/app/services/account.service.ts

```diff
@@ -11,26 +11,9 @@ export class AccountService {
 
   mockAccounts: Account[];
 
```

Comment:
Why is anything in this test being deleted?

Reply:
Sorry, I was under the impression that this was an anomaly from my team. If it's actually something that you guys put in, then definitely keep it.